test/e2e/resize_nodes.go

```diff
@@ -38,9 +38,12 @@ import (
 
 const serveHostnameImage = "gcr.io/google_containers/serve_hostname:1.1"
 
+const timeoutWaitNodeReady = 2 * time.Minute
+const timeoutWaitNodeNotReady = 2 * time.Minute
```

Comment:
Regrettably, because the `e2e` package is so giant, these constant names are available in **all** e2e tests, so they're not specific enough. 

E.g. in the reboot tests, we call these `rebootNodeNotReadyTimeout`, etc.